README.md

```diff
@@ -97,48 +80,54 @@ and to run one of the C#, Python, and Java console applications/scripts:
 2. Switch to `semantic-kernel` directory and then checkout experimental Java branch: `git checkout experimental-java`
```

Comment:
You can combine these first two steps:

`git clone -b experimental-java https://github.com/microsoft/semantic-kernel.git`